gen/build_deploy/powershell/dcos_generate_config_win.sh.in

```diff
@@ -8,6 +8,18 @@
 #
 set -o errexit -o nounset -o pipefail
 
+# check if dcos-config-win package exists:
+if [ -f ./genconf/serve/cluster-package-info.json ]; then
+    exists="$(grep -Po '"dcos-config-win":*' ./genconf/serve/cluster-package-info.json)";
+    if ! [ "$exists" ] ; then
```

Comment:
We never get to the `if` when the test fails. The failure of the `grep` causes us to exit before the `if`.

Whatever command you use to detect if the `dcos-config-win` package exists needs to be inside the `if` so that it is ignored by the shell.

e.g. `if ! [ $(grep -Po '"dcos-config-win":*' ./genconf/serve/cluster-package-info.json) ]; then`

Reply:
fixed